### DISCUSSION TITLE
[Bug]: Multiple database cache calls

### DISCUSSION THREAD

**Discussion by valentin-** [pimcore/pimcore#11000] [Q&A / Support]
### Expected behavior

Im trying to load a list of products in a Controller (Route), prefilter them using php and finally render them.

```
$productsListing = new \Pimcore\Model\DataObject\Product\Listing();
$productsListing->setCondition('productType = :type', [
    'type' => $category->getName(),
]);
$productsListing->setLimit(20);
$products = $productsListing->load(); 
```

Loading a bigger list on production fails, because of an database memory limit error.

The Symfony profiler showed, that there a many database queries selecting the same cache items - in this case 303 (with this id) of 1242 items total.

![image](https://user-images.githubusercontent.com/90033/145431549-a7040386-6316-4913-a54d-a4815aa89b4b.png)

Is this working as intended? Caching is enabled, latest Pimcore X version.

### Actual behavior

-

### Steps to reproduce

-

**Comment by fashxp** [comment]
isn't the first param the item_time? please check the second param. 

and besides that, please don't use DB as cache backend in production environments, use redis instead.
